api_docs/changelog.md

```diff
@@ -20,6 +20,20 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 193**
+
+* [`GET /messages`](/api/get-messages),
+  [`GET /messages/matches_narrow`](/api/check-messages-match-narrow),
+  [`POST /message/flags/narrow`](/api/update-message-flags-for-narrow),
+  [`POST /register`](/api/register-queue):
+  For [search/narrow filters](/api/construct-narrow) with the `id`
+  operator, added support for encoding the message ID operand as either
+  a string or an integer. Previously, only string encoding was supported.
+  Note that filters that accept stream or user IDs for the operand are
+  different. These operands must be encoded as integers or integer arrays,
+  because these filters expect either the stream name or user API emails
+  when the operand is a string.
```

Comment:
I don't think I follow this part. These operands must be encoded as integers or integer arrays, but they can also be strings?

It sounds like this isn't about something that changes at this feature level, though, so perhaps it can just be left out of this changelog entry (and perhaps it'd be useful to add something in the main text at /api/construct-narrow ). I think the first half of the paragraph is clear enough in saying that it's only talking about the `id` operator.

Reply:
I added that text when we were updating both operands, and I was probably over-thinking what needed to be explained here in the changelog entry.

There is a note above the example for the user / stream ID section about how they should be encoded as integers or integer arrays, so I think we're already covered as explicitly stating this on the `/api/construct-narrow` page.